actions/get_formatted_error.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# Copyright 2019 Encore Technologies
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from six import string_types
+from lib.base_action import BaseAction
+
+
+IGNORED_ERROR_TASKS = [
+    'send_error_email',
+    'provision_cleanup_exec'
+]
+
+
+class GetFormattedError(BaseAction):
+
+    def __init__(self, config):
+        """Creates a new BaseAction given a StackStorm config object (kwargs works too)
+        :param config: StackStorm configuration object for the pack
+        :returns: a new BaseAction
+        """
+        super(GetFormattedError, self).__init__(config)
+
+    def find_error_execution(self, parent_execution):
+        if hasattr(parent_execution, 'children'):
+            for m in parent_execution.children:
+                self.find_error_execution(m)
+        else:
+            st2_executions = self.st2_client.executions  # pylint: disable=no-member
+            execution = st2_executions.get_by_id(parent_execution)
+            if (execution.status == "failed" and
+               execution.context['orquesta']['task_name'] not in IGNORED_ERROR_TASKS):
+                self.parent_error = execution  # pylint: disable=no-member
+                if hasattr(execution, 'children'):
+                    for c in execution.children:
+                        self.find_error_execution(c)
+                else:
+                    self.child_error.append(execution)  # pylint: disable=no-member
+
+    def format_error(self, cmdb_request_item_url, cmdb_request_item):
+        err_string = ""
+        if cmdb_request_item_url:
+            err_string += ("ServiceNow Request: <a href={0}>{1}</a><br>"
+                            "".format(cmdb_request_item_url, cmdb_request_item))
+
+        if self.child_error:  # pylint: disable=no-member
```

Comment:
I think if you initialize these variables in the init function you wouldn't need `# pylint: disable=no-member`